air-script/tests/bitwise/bitwise.air

```diff
@@ -0,0 +1,56 @@
+def BitwiseAir
+
+public_inputs:
+    stack_inputs: [16]
+
+trace_columns:
+    main: [s, a, b, a0, a1, a2, a3, b0, b1, b2, b3, zp, z, dummy]
+
+periodic_columns:
+    k0: [1, 0, 0, 0, 0, 0, 0, 0]
+    k1: [1, 1, 1, 1, 1, 1, 1, 0]
+
+boundary_constraints:
+    # This is a dummy trace column to satisfy requirement of atleast one boundary constraint.
+    enf dummy.first = 0
+
+transition_constraints:
+    # Enforce that selector must be binary
+    enf s^2 - s = 0
+
+    # Enforce that selector should stay the same throughout
+    enf k1 * (s' - s) = 0
+
+    # Enforce that input is decomposed into valid bits
+    enf a0^2 - a0 = 0
+    enf b0^2 - b0 = 0
+    enf a1^2 - a1 = 0
+    enf b1^2 - b1 = 0
+    enf a2^2 - a2 = 0
+    enf b2^2 - b2 = 0
+    enf a3^2 - a3 = 0
+    enf b3^2 - b3 = 0
```

Comment:
nit: I would prefer to group the `a` bits together and the `b` bits together. It's obviously not so significant in an example, but if we use this for our actual constraints it would be much clearer to do it that way, and we are likely to use this as a basis for our constraints in the future